client/client.lua

```diff
@@ -437,6 +527,29 @@ local function CopyToClipboard(dataType)
             string = h
         })
         QBCore.Functions.Notify(Lang:t("success.heading_copied"), "success")
+    elseif dataType == 'freeaimEntity' then
+        local entity = GetFreeAimEntity()
+
+        if entity then
+            local entityHash    = GetEntityModel(entity)
+            local entityName    = Entities[entityHash] ~= nil and Entities[entityHash] or "Unknown"
```

Comment:
```suggestion
local entityName    = Entities[entityHash] or "Unknown"
```
No need of a "non nil" condition for this!